src/backend/partaj/core/migrations/0044_auto_20210521_1125.py

```diff
@@ -0,0 +1,93 @@
+# Generated by Django 3.0.5 on 2021-05-21 11:25
```

Comment:
May I suggest giving a name to this migration? All existing migrations have a name that summarizes what they do.

You can do this with a flag on `makemigrations`: eg. `--name "add_referral_message_model"` for the previous migration